docs/apm/service-overview.asciidoc

```diff
@@ -40,6 +42,10 @@ The time comparison options are based on the selected time filter range:
 |An identical amount of time immediately before the selected time range
 |====
 
+You can use expected bounds comparison if {ml-jobs} exist in your selected 
```

Comment:
"the expected bounds"? There's a the in front of this phrase earlier in the doc.

Reply:
Thanks! Fixed in [7cf6229](https://github.com/elastic/kibana/pull/134472/commits/7cf6229f3627dac94849c737bf8572c3d81624fd).